vrc-get-gui/app/_main/packages/templates/index.tsx

```diff
@@ -678,7 +683,10 @@ function TemplateEditor({
 		const templateInfo = templates.find((x) => x.id === baseTemplate);
 		if (templateInfo == null) return [];
 		// unityVersions is in order
-		const unityVersions = templateInfo.unity_versions;
+		// currently, ignore the unity version channel part and increment part
+		const unityVersions = templateInfo.unity_versions.map(
+			(x) => x.split(/[^\d.]/, 2)[0],
```

Comment:
```suggestion
			(unity_version) => unity_version.split(/[^\d.]/, 2)[0],
```
可読性が少し上がります(読めば分かる範囲ではあります)
深く考えずにxにしていたので、必要であれば